versions/app/src/screens/menuDetailScreen.js

```diff
@@ -178,14 +179,12 @@ export default function MenuDetailScreen({ route, navigation }) {
 				<View style={styles.content}>
 					<View style={styles.header}>
 						<View style={styles.title}>
-							<Text style={globalStyles.h1}>{details.title}</Text>
+								<Text style={globalStyles.h1}>{details.title}</Text>
+						</View>
+						<View style={styles.price_cont}>
+								<Text style={styles.price_label}>EPC Suggested Price</Text>
```

Comment:
I think "Suggested Price" is better than "EPC Suggested Price". I think that it's suggested by EPC is implied since they're using the EPC service.

Reply:
Thanks for the feedback. I took a random mockup from online by searching pricing UI/UX. That makes sense that the user already knows that they are using EPC and not a service that aggregates like a 3rd party.